deny.toml

```diff
@@ -0,0 +1,180 @@
+# This template contains all of the possible sections and their default values
```

Comment:
Update the comment (this is not a template any more) and just keep a high level description, and a link to https://embarkstudios.github.io/cargo-deny/checks/cfg.html

Reply:
Done.